x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/setup_technology.ts

```diff
@@ -15,6 +15,20 @@ import { SelectedPolicyTab } from '../../components';
 
 export const AGENTLESS_POLICY_ID = 'agentless';
 
+export const useAgentlessPolicy = () => {
+  const { agentless: agentlessExperimentalFeatureEnabled } = ExperimentalFeaturesService.get();
+  const { cloud } = useStartServices();
+  const isServerless = cloud?.isServerlessEnabled ?? false;
```

Comment:
a bit of a nit comment just from a javascript perspective, I find `??` to be a bit weird since it reads as though we expect to get `0` or `''` and use them as valid values. As far as i can see the type of `isServerless` is a boolean, so in this case i suggest using

```suggestion
  const isServerless = !!cloud?.isServerlessEnabled;
```

would ensure the value is always a boolean value